### PR TITLE
Config map fixes

### PR DESCRIPTION
**Pull request by jgruica**

### CODE REVIEW

File: pkg/store/s3pg/version_store.go [hunk 1]
```diff
@@ -51,27 +51,46 @@ func (s S3PGStore) GetClientset() (*kubernetes.Clientset, error) {
...
+
+	if err := s.updateConfigmap(existingConfigMap); err != nil {
+		return errors.Wrap(err, "failed to update metadata configmap")
 	}
 
-	return existingConfigmap, nil
+	return nil
 }
 
 func (s S3PGStore) updateConfigmap(configmap *corev1.ConfigMap) error {
```
Comment (divolgin) [inline pkg/store/s3pg/version_store.go:96]: Get rid of this function and inline its code

File: pkg/store/s3pg/version_store.go [hunk 2]
```diff
@@ -51,27 +51,46 @@ func (s S3PGStore) GetClientset() (*kubernetes.Clientset, error) {
...
+		}
+
+		if createdConfigMap.Data == nil {
+			createdConfigMap.Data = map[string]string{}
 		}
-		return createdConfigmap, nil
+
+		createdConfigMap.Data["application.yaml"] = applicationMetadata
+
+		if err := s.updateConfigmap(createdConfigMap); err != nil {
```
Comment (divolgin) [inline pkg/store/s3pg/version_store.go:78]: this config was just created.  is this update needed?

Comment (jgruica) [reply pkg/store/s3pg/version_store.go:78]: not needed

File: pkg/store/s3pg/version_store.go (hunk 1 above)
Comment (jgruica) [reply pkg/store/s3pg/version_store.go:96]: get rid of what function?

File: pkg/store/s3pg/version_store.go (hunk 2 above)
Comment (jgruica) [reply pkg/store/s3pg/version_store.go]: updated!

**Review by divolgin — APPROVED** [review]

**Pull request merged**